### PR TITLE
rel: prep 2.7 release

### PR DESCRIPTION
**Pull request by kentquirk**
Prep 2.7 release

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,9 +2,43 @@
...
+Instead, it is used as a more general publish/subscribe framework for rapidly sharing information between nodes in the cluster.
+Things that are shared with this connection are:
+
+- Peer membership
+- Stress levels
+- News of configuration changes
+
+Because of this mechanism, Refinery will now react more quickly to changes in any of these factors.
+In particular, all of the nodes in the cluster will now go into and out of stress relief together.
+Also, when one node detects a configuration change, all of its peers will be told about it immediately.
```
Comment (VinozzZ) [inline RELEASE_NOTES.md:20]: Do we want to add a disclaimer about the delay introduced in k8s if configurations are mounted as a ConfigMap?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,9 +2,43 @@
...
+The metric called `stress_level` has been split into `individual_stress_level` (the stress level as seen by a single node) and `cluster_stress_level` (the aggregated cluster level). If you were using the `stress_level` metric in SLOs, boards or triggers, you must update as the metric is no longer being sent.
+
+There is also a new pair of metrics, `config_hash` and `rule_config_hash`.
+These are numeric Gauge metrics that are set to the numeric value of the last 4 hex digits of the hash of the current config files.
+These can be used to track that all refineries are using the same configuration file.
+
+### Disabling Redis and using a static list of peers
+Specifying `PeerManagement.Type=file` will cause Refinery to use the fixed list of peers found in the configuration.
+This means that Refinery will operate without sharing changes to peers, stress, or configuration, as it has in previous releases.
+
```
Comment (VinozzZ) [inline RELEASE_NOTES.md:38]: Maybe we also need to mention the Health check change?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -41,7 +41,8 @@ In this release, this will only happen when a Refinery node is shutting down.
 ### Metrics changes
 There have also been some minor changes to metrics in this release:
 
-The metric called `stress_level` has been split into `individual_stress_level` (the stress level as seen by a single node) and `cluster_stress_level` (the aggregated cluster level). If you were using the `stress_level` metric in SLOs, boards or triggers, you must update as the metric is no longer being sent.
+We hae two new metrics called `individual_stress_level` (the stress level as seen by a single node) and `cluster_stress_level` (the aggregated cluster level).
```
Comment (VinozzZ) [inline RELEASE_NOTES.md:44]: `hae` -> `have`

**Review by VinozzZ — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,9 +2,65 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.7.0
+
+This release is a transitional release, laying the groundwork for substantial future changes to Refinery.
```
Comment (cartermp) [inline RELEASE_NOTES.md:7]:
```suggestion
This release is a minor release focused on better cluster stability and data quality with a new system for communicating peer information across nodes.
As a result, clusters should generally behavior more consistently.

Refinery 2.7 lays the groundwork for substantial future changes to Refinery.
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -2,9 +2,65 @@
...
+- News of configuration changes
+
+Because of this mechanism, Refinery will now react more quickly to changes in any of these factors.
+When one node detects a configuration change, all of its peers will be told about it immediately.
+
+In addition, Refinery now publishes individual stress levels between peers.
+Nodes calculate a cluster stress level as a weighted average (with nodes that are more stressed getting more weight).
+If an individual node is stressed, it can enter stress relief individually.
+This may happen, for example, when a single giant trace is concentrated on one node.
+If the cluster as a whole is being stressed by a general burst in traffic, the entire cluster should now enter or leave stress relief at approximately the same time.
```
Comment (cartermp) [inline RELEASE_NOTES.md:25]: Does this mean that you could expect to see a rise in cross-node network traffic, or do we expect it to be marginal?

**Review by cartermp — COMMENTED** [review]
Recommended copyedit

File: RELEASE_NOTES.md (hunk 5 above)
Comment (kentquirk) [reply RELEASE_NOTES.md:28]: You might see the impact of pubsub traffic to and from redis, but it's pretty small; we mostly only publish new values when they're different.

**Review by mjingle — APPROVED** [review]
Looks great to me wrt docs! 🎉 Thank you @kentquirk!

**Pull request merged**